### ISSUE TITLE
ProcDqCs readout is wrong in Agesa 1.2.0.0 and newer

### ISSUE THREAD

**Issue by lextra2** [irusanov/ZenTimings#50]
After updating to a bios with Agesa 1.2.0.0 or newer, `ProcDqCs` is being readout as `Hi-Z` which I'm pretty sure is incorrect.

Some people speculated that on newer Agesas, `ProcDqCs` is set to a lower Ohm value than previously, and that value isn't supported in ZenTimings list, and so ZenTiming, instead of going further down that list, ends up picking the first value of that list, which is Hi-Z

tldr: Add lower Ohm values to the supported list of `ProcDqCs`

**Comment by irusanov** [comment]
Can you test with the new release if there's a difference? Also a debug log could be helpful. Thanks!

**Comment by irusanov** [comment]
I do have an account there and also a dedicated thread for zentimings.

**Comment by lextra2** [comment]
I think this is a AGESA and stability issue.
ZenTimings readout is correct.
If you're getting Hi-Z on "Auto", rollback bios or set ProcDqCs manually to 34.3
